helm_deploy/hmpps-book-video-link/values.yaml

```diff
@@ -3,3 +3,6 @@
 
 generic-prometheus-alerts:
   targetApplication: hmpps-book-video-link
+
+  ingress:
```

Comment:
ingress top level and this inside and annotations section

Reply:
fixed